app/assets/stylesheets/katello/katello.scss

```diff
@@ -1,5 +1,4 @@
 /**
- * = require "katello/jquery.loadmask.css"
```

Comment:
i still see this here: https://github.com/Katello/katello/blob/master/vendor/assets/stylesheets/katello/jquery.loadmask.css.scss

Can you explain a bit more?

Reply:
Hrm, that's correct... unfortunately I have no idea thean...

Reply:
Hrm, that's correct... unfortunately I have no idea thean...